aggregator/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM denoland/deno:1.20.6
+FROM denoland/deno:1.23.4
```

Comment:
This is necessary to make new builds work, due to the everpresent dastardly moving targets that are our dependencies.